app/view_models/nsm/v1/core_cost_summary.rb

```diff
@@ -51,7 +51,7 @@ def sum_allowed(data, field)
       def calculate_profit_costs(formatted:)
         letters_calls = LettersAndCallsSummary.new('submission' => submission).rows
 
-        calculate_row(work_items[PROFIT_COSTS] || ([] + letters_calls), 'profit_costs', formatted:)
+        calculate_row((work_items[PROFIT_COSTS] || []) + letters_calls, 'profit_costs', formatted:)
```

Comment:
This was a bug preventing letters and calls from  being included in profit costs